frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -89,19 +103,9 @@ const footerVariantStyles = {
   `,
 };
 
-export const ParametersWidgetContainer = styled.div`
-  display: flex;
-  align-items: flex-start;
-
-  padding: 0.5rem;
-
-  ${breakpointMinSmall} {
-    padding: 0.5rem 1rem 1rem 1rem;
-  }
-
-  ${breakpointMinLarge} {
-    padding: 0.5rem 1.5rem 1.5rem 1.5rem;
-  }
+export const ParametersWidgetContainer = styled(FullWidthContainer)`
```

Comment:
I found one minor issue. This component still takes up space when we don't have visible parameters.
![image](https://github.com/metabase/metabase/assets/1937582/9c7af1b9-6fa8-49ca-8a05-12fd99dbe4a2)

This doesn't happen to normal dashboards though.

Reply:
Since this was happening previous to this PR, we will address this in a follow-up PR.

Thank you for catching this, Kelvin!